canonical/store-components
# A11y fixes

## Done
- Add logic to prevent empty heading tags in cards
- Make the title of topic cards linked

## QA
- Go to https://store-components-55.demos.haus/?path=/story/topiccard--default-story  and check that the title is linked
- Go to https://store-components-55.demos.haus/?path=/story/defaultcard--no-heading and check that there is no heading
